src/main/java/io/github/thebusybiscuit/cscorelib2/data/PersistentDataAPI.java

```diff
@@ -89,6 +91,13 @@ public static void setJsonArray(PersistentDataHolder holder, NamespacedKey key,
         holder.getPersistentDataContainer().set(key, PersistentJsonDataType.JSON_ARRAY, value);
     }
 
+    public static void setUuid(PersistentDataHolder holder, Plugin plugin, String key, UUID uuid) {
```

Comment:
Can you please add javadocs and parameter annotations here?

Reply:
Sure, hadn't added anything as nothing else has it (no docs for the setters, no annotations for any parameter). I'll add mine and *maybe* if I want to bore myself to death PR an addition to the others